src/Sql/Sql.Test/ScenarioTests/ManagedInstanceFailoverScenarioTests.cs

```diff
@@ -41,21 +41,21 @@ public void TestFailoverManagedInstance()
             RunPowerShellTest("Test-FailoverManagedInstance");
         }
 
-        [Fact]
+        [Fact(Skip = "Depends on hardcoded resource")]
```

Comment:
We need to ge a positive confirmation from the test owner (or other knowledgeable dev) that this behavior has not regressed.

Reply:
Signing off on this test - @MilanBrkicFON will follow up to make the test not depend on hardcoded resources.

Reply:
same question.  need signoff for all the newly skipped tests in this file

Reply:
The whole file.